packages/SwingSetRunner/src/main.js

```diff
@@ -0,0 +1,51 @@
+
+import process from 'process';
+import repl from 'repl';
+import util from 'util';
+
+import { loadBasedir, buildVatController, buildLoopbox } from '@agoric/swingset-vat';
+
+function deepLog(item) {
+  console.log(util.inspect(item, false, null, true));
+}
+
+export async function main() {
+  let argv = process.argv.splice(2);
+  let withSES = true;
+  if (argv[0] === '--no-ses') {
+    withSES = false;
+    argv.shift();
+  }
+  const command = argv.shift();
+  if (command !== 'run' && command !== 'shell') {
+    throw new Error(`use 'vat run' or 'vat shell', not 'vat ${command}'`);
+  }
+  const basedir = (argv[0] === '--' || argv[0] === undefined) ? '.' : argv.shift();
+  const vatArgv = argv[0] === '--' ? argv.slice(1) : argv;
+
+  const config = await loadBasedir(basedir);
+  const lb = buildLoopbox();
```

Comment:
Let's omit the `loopbox` device, at least for now. It's used by one of the demos that exercises the comms layer, but we can get the basic work done with this new repo without any comms support. (and even then it's used to fake a loopback channel, so it's not clear that a comms-capable runner tool should be using it either).